app/assets/stylesheets/components/_modal.scss

```diff
@@ -35,14 +35,14 @@
     position: relative;
 
     &.key-badge::before {
-      background-image: url('p-key.svg');
```

Comment:
Can we remove the `p-key.svg` image?

Reply:
Gone as of [cf6cb0e](https://github.com/18F/identity-idp/pull/6983/commits/cf6cb0e4b84f88dc178f730fa31ceb2d05eb9582)